src/pages/productDetails.tsx

```diff
@@ -135,6 +142,7 @@ export const getServerSideProps = (ctx: NextPageContext): { props: ProductDetail
         props: {
             product: !productDetailsCookie ? {} : JSON.parse(productDetailsCookie),
             operator: operatorInfo.operator,
+            passengerType: passengerTypeInfo.passengerType,
```

Comment:
could we destructure this like `const {passengerType} =  passengerTypeInfo `. Then in the props can just have passengerType?

Reply:
Added destructuring for 'passengerType'. Did the same for 'operator' for consistency's sake.